ui-kit/src/main/res/menu/menu_ui_kit_bottom_navigation.xml

```diff
@@ -3,22 +3,22 @@
     xmlns:android="http://schemas.android.com/apk/res/android">
     <item
         android:id="@+id/homeItem"
-        android:icon="@drawable/ic_bottom_navigation_wallet"
+        android:icon="@drawable/ic_wallet_main"
         android:title="@string/common_wallet" />
     <item
         android:id="@+id/historyItem"
-        android:icon="@drawable/ic_bottom_navigation_history"
+        android:icon="@drawable/ic_history_main"
```

Comment:
it would be better to name all icons with this naming
ic_[place]_[whats on the icon]
so it would be 
ic_main_history
ic_main_wallet
ic_main_feedback
ic_main_settings